### ISSUE TITLE
what not to commit 

### ISSUE THREAD

**Issue by osamaAbumansour** [wafa200099/Library-Management-System-Database-Project-#2]
avoid committing files unrelated to the project itself. for example the .idea folder should not be committed. read on git ignore and exclude

**Comment by wafa200099** [comment]
Yes its My bad I commit it By mistake and then I was lazy to remove 
thanks
